kvmagent/kvmagent/test/unittest_tools/unittest_pypi_source/install.sh

```
@@ -1,2 +1 @@
-pip install -r requirements/requirements1.txt -i file://`pwd`/pypi/simple
-pip install -r requirements/requirements2.txt -i file://`pwd`/pypi/simple
+pip install -r requirements/requirements1.txt -i file://`pwd`/pypi/simple
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)